src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.cs

```diff
@@ -2461,7 +2461,12 @@ public virtual ToolStripItem GetNextItem(ToolStripItem start, ArrowDirection dir
                 case ArrowDirection.Right:
                     return GetNextItemHorizontal(start, forward: true);
                 case ArrowDirection.Left:
-                    return GetNextItemHorizontal(start, forward: false);
+                    bool forward = false;
+                    if ((LastKeyData == (Keys.Shift | Keys.Tab) && RightToLeft == RightToLeft.No) || (LastKeyData == Keys.Tab && RightToLeft == RightToLeft.Yes))
```

Comment:
🤔  Would this increase readability?
```suggestion
                    bool isRtl = RightToLeft == RightToLeft.Yes;
                    bool forward = (LastKeyData == (Keys.Shift | Keys.Tab) && !isRtl) || (LastKeyData == Keys.Tab && isRtl)
```

Reply:
Thanks, I changed the implementation.